.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+
+name: TABTestKit CI
+
+on:
+  workflow_dispatch: # This allows you to manually run the workflow from GitHub's web UI
+  pull_request:
+    branches: [ '*' ] # This means this workflow will run when any PR is created or any changes are pushed to a PR, on any branch
+
+jobs:
+  test_xcode10_ios12:
+    name: Run tests on Xcode 10 and iOS 12
+    runs-on: macOS-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set Xcode version to 10.3
+        run: sudo xcode-select -switch /Applications/Xcode_10.3.app
+      - name: Build for testing
+        run: xcodebuild build-for-testing -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit-Example -destination 'platform=iOS Simulator,name=iPhone Xs,OS=12.4'
+      - name: Test on iPhone Xs
+        run: xcodebuild test-without-building -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit-Example -destination 'platform=iOS Simulator,name=iPhone Xs,OS=12.4'
+
+  test_xcode11_ios13:
+    name: Run tests on Xcode 11 and iOS 13
+    runs-on: macOS-latest
```

Comment:
Will be better to specify the version to Catalina?

Reply:
Xcode 11 can run on Big Sur too, so when that's available we'll probably want CI to run on that, unless you want to create specific jobs for different versions of macOS?

Reply:
When Big Sur will be available, do we want to run on the first release? I haven't play with macOS Big Sur to know how stable it is. Especially they bumped to version 11 so there is a risk that some dependencies are broken with the version check on 10.x.

Reply:
We should do yeah, since users of our library can update to Big Sur whenever they want, so we should run our CI against it too :) if it turns out to be a problem, we can fix whatever is broken or recommend people don't update, but we should always be aiming for the latest versions of everything since it's a public repository

Reply:
It's definitely better for us to find out asap that there's a problem with our repository on a new version of macOS than wait for users to report it to us